nixos/modules/services/x11/display-managers/gdm.nix

```
@@ -147,6 +147,9 @@ in
 
     services.displayManager =
       {
+        # Enable desktop session data
+        enable = lib.mkDefault true;
```

```suggestion
        enable = true;
```
otherwise things don't work as you already seen

---

What do you mean by "as you already seen" ? Something else is disabling `displayManager` ?
`greetd` set it to `mkDefaul true` too.

---

That the generated service then does not work.

I think it is false over there, too.

---

Ok you are surely right. I have set it to `true`. I don't see any reason to enable `gdm` but wanting `services.displayManager` disabled.